File: Izzy.Web/Client/src/store/model/Person.ts
import { Rouble } from './Rouble';

export class Person {
  protected _name: string;
  protected _roubles: Rouble;

  constructor(name: string, roubles: Rouble) {
    this._name = name;
    this._roubles = roubles;
  }

  public name(): string {
    return this._name;
  }

  public money(): number {
    return this._roubles.value();
  }
}

File: Izzy.Web/Model/Person.cs
using System;
using Newtonsoft.Json;

namespace Izzy.Web.Model
{
    public class Person
    {
        [JsonProperty("name")]
        public String Name { get; set; }

        [JsonProperty("roubles")]
        public Decimal Roubles { get; set; }

        public Person() {}

        public Person(String name, Decimal roubles)
        {
            this.Name = name;
            this.Roubles = roubles;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
